### PR TITLE
PTP CI adding more logs for system clock

### PR DESCRIPTION
**Pull request by edcdavid**

### CODE REVIEW

**Comment by edcdavid** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-ptp

**Comment by edcdavid** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.11-e2e-telco5g-ptp

File: ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh [hunk 1]
```diff
@@ -148,6 +148,13 @@ retry_with_timeout() {
   done
 }
 
+# print RTC logs
+print_time() {
+  oc debug node/cnfdf30.telco5gran.eng.rdu2.redhat.com -- chroot /host sh -c "date;sudo hwclock"
+  oc debug node/cnfdf31.telco5gran.eng.rdu2.redhat.com -- chroot /host sh -c "date;sudo hwclock"
+  oc debug node/cnfdf32.telco5gran.eng.rdu2.redhat.com -- chroot /host sh -c "date;sudo hwclock"
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh:155]: Can we probably discover nodes and loop over them? We'll need to fix every time if nodes names are changed, or different infrastructure will be set up.

Comment (edcdavid) [reply ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh]: fixed

File: ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh [hunk 2]
```diff
@@ -148,6 +148,18 @@ retry_with_timeout() {
   done
 }
 
+# print RTC logs
+print_time() {
+# Get the list of nodes in the cluster
+NODES=$(kubectl get nodes -l node-role.kubernetes.io/worker -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}')
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh:154]: Can you use `oc` instead of kubectl? We install `oc` explicitly.

**Pull request merged**